README.md

```diff
@@ -93,7 +93,8 @@ Run `mint help` to see usage instructions.
 - **run**: Runs a package. This will install it first if it isn't already installed, though won't link it globally. It's useful for running a certain version.
 - **list**: Lists all currently installed packages and versions.
 - **uninstall**: Uninstalls a package by name.
-- **bootstrap**: Installs all the packages in your [Mintfile](#mintfile) without linking them globally
+- **bootstrap**: Installs all the packages in your [Mintfile](#mintfile) without linking them globally.
+- **dump**: Creates a [Mintfile](#mintfile) from all the installed packages.
```

Comment:
Looking at it now, do you think have a `--dump` or `--mintfile` command on list would make more sense?

Reply:
I agree, "list --dump" is maybe more appropriate of the usage. The dump functionality is not a main feature in my opinion, that's why I put it under bootstrap initially. So I guess I will move it under list? :)

Reply:
I changed this to `mint list --dump`